MAUI/Button/Visual-States.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: Visual states in .NET MAUI Button Control | Syncfusion
+description: Learn here about visual states support in the Syncfusion .NET MAUI Button (SfButton) control, its elements and more.
+platform: maui
+control: SfButton
+documentation: ug 
+
+---
+
+# Visual States in .NET MAUI Button (SfButton)
+
+The visual of the Button can be customized using `VisualStates`. The `SfButton `control contains the following four visual states:
+
+* Normal
+* Pressed
+* Checked
+* Diabled
+
+N> The visual states Checked and Unchecked are only updated when enabling the IsCheckable property in SfButton.
```

Comment:
N> The visual states Checked and Unchecked are only updated when enabling the IsCheckable property in the SfButton.